### DISCUSSION TITLE
I need help to setup our connection of SnowFlake With AirFlow

### DISCUSSION THREAD

**Discussion by pedrovct97** [apache/airflow#29043] [Q&A]
If someone could help me, I'll appreciate it so much, if the person who help us is from Brazil, we can pay for it.

**Comment by hussein-awala** [comment]
Could this screenshot help you?

<img width="855" alt="renditionDownload" src="https://user-images.githubusercontent.com/21311487/218287572-8f8b0693-7317-436a-b259-58961916a42f.png">
